src/utils/bivariate/types/stat.types.ts

```diff
@@ -17,12 +12,14 @@ export type Step = {
   value: number;
 };
 
-/* Divisor and denominator pair */
+/* Nominator and denominator pair */
+type Unit = { id: string | null; longName: string | null; shortName: string | null };
 export type Quotient = [string, string]; // this field will be removed in next tickets, only Quotients will stay
 export type Quotients = {
   name: string;
   label: string;
   direction: Direction;
+  unit: Unit;
```

Comment:
are we certain it can't be `undefined`? as you wrote "we don't have nominator unit" before, even though it's about `unit.id` i should still check

Reply:
yeah, it always exists, checked this point